Python/Heart Disease Prediction/README.md

```diff
@@ -0,0 +1,23 @@
+# Heart disease prediction
+This python script uses a machine learning model to predict whether a person has heart disease or not.The Model is trained with [ucl heart disease dataset](https://archive.ics.uci.edu/ml/datasets/heart+disease)
+
+# How to run program
+**1.**  Fork [this](https://github.com/HarshCasper/Rotten-Scripts) repository.
+
+**2.**  Clone your forked copy of the project.
+```
+git clone https://github.com/<your-user-name>/Rotten-Scripts.git
+```
+Navigate to the project directory.
+```bash
+cd \Rotten-Scripts\Python\Heart Disease Prediction
+```
+Install the requirements 
+```bash
+pip install -r requirements.txt
+```
+## To run
+```bash
+python script.py
+```
+#### By [Adithya Kurien Peter]("https://github.com/aaadddiii")
```

Comment:
Is this correct?

Reply:
It works 
![](https://i.postimg.cc/fRR4fJHR/pic2.png)

Reply:
The Author line, is that correct?

Reply:
It was a mistake, sorry . I've made necessary changes